server/src/calls.ts

```diff
@@ -1,5 +1,5 @@
 
-import tsp from 'typescript/lib/protocol';
+import * as tsp from 'typescript/lib/protocol';
```

Comment:
This was showing up as an error in my editor.

Reply:
Maybe because you haven't run `yarn` after rebasing? It shouldn't be needed AFAIK.

Reply:
I think vscode hadn't picked up the tsconfig change. `"esModuleInterop": true` means it should work.